deps/spidershim/tests.gyp

```diff
@@ -28,5 +28,9 @@
       'target_name': 'value',
       'sources': [ 'test/value.cc' ],
     },
+    {
+      'target_name': 'v8v8',
+      'sources': [ 'test/v8v8.cc' ],
```

Comment:
Please add this to [run-tests.sh](https://github.com/mozilla/spidernode/blob/master/deps/spidershim/scripts/run-tests.sh#L12) too.  (We don't have a fully functioning build system yet, so we need to manually get the tests to run on Travis-CI!)